go.mod

```diff
@@ -468,3 +468,5 @@ require (
 	kernel.org/pub/linux/libs/security/libcap/psx v1.2.73 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
```

Comment:
We need to publish a new version of the terraform provider before we can merge this PR, so that we can get rid of this `replace`.

Reply:
Is it okay to merge with this replace?

https://github.com/coder/terraform-provider-coder/commit/129c295afed8 - this commit was out of the tree.
I had to create `yevhenii-prebuilds` branch which points to this commit just to make sure it builds.

Reply:
We've merged the necessary changes in the coder terraform provider.
I've updated the dependency in this project to point to the appropriate commit in mainline. I think we can merge this PR now, and update the go.mod file again once we release the new version of the terraform provider with a tag that includes our changes.